var/spack/repos/builtin/packages/py-coloredlogs/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyColoredlogs(PythonPackage):
+    """Colored terminal output for Python's logging module"""
+
+    homepage = "https://pypi.org/project/coloredlogs/"
+    url      = "https://files.pythonhosted.org/packages/63/09/1da37a51b232eaf9707919123b2413662e95edd50bace5353a548910eb9d/coloredlogs-10.0.tar.gz"
```

Comment:
```suggestion
    url      = "https://pypi.io/packages/source/c/coloredlogs/coloredlogs-10.0.tar.gz"
```

See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#url

Reply:
Still need to fix URL

Reply:
Sorry thought I had done that, but it wasn't pushed.